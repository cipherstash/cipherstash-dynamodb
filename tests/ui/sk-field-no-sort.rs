use cipherstash_dynamodb::Encryptable;

#[derive(Debug, Encryptable)]
struct User {
    #[cryptonamo(query = "exact", compound = "email#name")]
    #[partition_key]
    email: String,
    #[cryptonamo(query = "prefix", compound = "email#name")]
    name: String,

    sk: String,
}

fn main() {}
