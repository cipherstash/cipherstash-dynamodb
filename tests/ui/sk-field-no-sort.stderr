error: field named 'sk' must be annotated with #[sort_key]
  --> tests/./ui/sk-field-no-sort.rs:11:5
   |
11 |     sk: String,
   |     ^^^^^^^^^^
